impl/src/main/java/org/ehcache/impl/internal/store/offheap/AbstractOffHeapStore.java

```diff
@@ -233,6 +235,7 @@ public Number call() throws Exception {
           }
 
           if (updateAccess) {
+            mappedValue.forceDeserialization();
```

Comment:
I wonder if this method should be named `detach`... or something similar... thats what this does right?